types/trace/trace.go

```diff
@@ -42,10 +42,18 @@ type Event struct {
 	Syscall               string       `json:"syscall"`
 	StackAddresses        []uint64     `json:"stackAddresses"`
 	ContextFlags          ContextFlags `json:"contextFlags"`
-	Args                  []Argument   `json:"args"` // Arguments are ordered according their appearance in the original event
+	EntityID              uint32       `json:"-"`    // task unique identifier (*)
```

Comment:
Lets see if having the current task hash is enough to maintain the tree (instead of having parent starttime and parent hash).